client/src/styles/login-form.scss

```diff
@@ -217,17 +217,17 @@ input[type="checkbox"] {
   @each $pos in $positions {
     &.bs-popover-auto[x-placement^="#{$pos}"],
     &.bs-popover-#{$pos} {
-      > .arrow:before {
+      > .popover-arrow:before {
         border-#{$pos}-color: var(--color-field-border);
       }
 
-      > .arrow:after {
+      > .popover-arrow:after {
         border-#{$pos}-color: var(--color-form-background);
       }
     }
   }
 
-  .popover-header + .popover-body {
+  .popover-body {
```

Comment:
`.arrow` is now `.popover-arrow` as per https://getbootstrap.com/docs/5.0/migration/#popovers

The `.popover-header` element is no longer present.